js/archive.js

```diff
@@ -44,7 +53,7 @@
     listing.innerHTML = ''
   }
 
-  function getShows(page, search) {
+  function getShows(page, search = '') {
```

Comment:
https://caniuse.com/#feat=mdn-javascript_functions_default_parameters

it's gonna be nice when we give ourselves a build step